manual-install/latest.yml

```diff
@@ -38,11 +39,11 @@ services:
       - POSTGRES_USER=nextcloud
       - TZ=${TIMEZONE}
       - PGTZ=${TIMEZONE}
-    stop_grace_period: 1800s
+    stop_grace_period: 1800
```

Comment:
@Zoey2936 do you know if it also works without the s unit or will docker-compose bug out?

Reply:
apparently here it is indeed needed.